site/_docs/configuration.md

```diff
@@ -571,11 +571,11 @@ file or on the command-line.
 # Where things are
 source:       .
 destination:  ./_site
-plugins_dir:  ./_plugins
-layouts_dir:  ./_layouts
-data_dir:     ./_data
-includes_dir: ./_includes
-collections:  null
+plugins_dir:  _plugins
+layouts_dir:  _layouts
+data_dir:     _data
+includes_dir: _includes
```

Comment:
They're resolved using `Jekyll.sanitized_path(site.source, the_dir)` so it shakes out to be the same. :shrug: